pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -125,6 +125,12 @@ var (
 		Name: "pilot_xds_push_errors",
 		Help: "Number of errors (timeouts) pushing to sidecars.",
 	}, []string{"type"})
+
+	proxiesConvergeDelay = prometheus.NewHistogram(prometheus.HistogramOpts{
+		Name:    "pilot_proxy_convergence_time",
+		Help:    "Delay between config change and all proxies converging.",
+		Buckets: []float64{.005, .01, .1, 1, 10},
```

Comment:
I think we need buckets between 1 and 10s. There is meaningful difference in those buckets.
perhaps `{0.005, 0.01, 0.1, 1, 3, 5, 10}`

Reply:
Done, also 30s per @costinm suggestion.